### DISCUSSION TITLE
pipeline distortion calibration

### DISCUSSION THREAD

**Discussion by moolight-seashell** [alicevision/Meshroom#2272] [General]
i have read the docs and im not sure how to use this because as it say this is not connected to the main pipeline, 

what we ar supposed to copy where ?

in any case this isn't worksing for me
![image](https://github.com/alicevision/Meshroom/assets/49676861/42229390-5592-4924-adea-89083cf94436)

(i have 25 calibartion images here)

**Comment by natowi** [comment]
What are your settings in the DistortionCalibration node?
